pkg/configurator/methods.go

```diff
@@ -193,3 +193,18 @@ func (c *Client) GetInboundExternalAuthConfig() auth.ExtAuthConfig {
 
 	return extAuthConfig
 }
+
+// IsWASMStatsEnabled returns whether the WASMStats feature flag is enabled
+func (c *Client) IsWASMStatsEnabled() bool {
+	return c.getMeshConfig().Spec.FeatureFlags.EnableWASMStats
+}
+
+// IsEgressPolicyEnabled returns whether the EgressPolicy feature flag is enabled
+func (c *Client) IsEgressPolicyEnabled() bool {
+	return c.getMeshConfig().Spec.FeatureFlags.EnableEgressPolicy
+}
+
+// IsMulticlusterModeEnabled returns whether the MultiClusterMode feature flag is enabled
+func (c *Client) IsMulticlusterModeEnabled() bool {
+	return c.getMeshConfig().Spec.FeatureFlags.EnableMulticlusterMode
+}
```

Comment:
I think it would make sense to collapse these into a single struct defined in `pkg/featureflags`:
```
type struct Feature {
    WASMStats bool
    EgressPolicy bool
    Multicluster bool
}
```